titus-ext/jooqflyway/src/main/java/com/netflix/titus/ext/jooqflyway/jobactivity/RdsUtils.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2020 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.titus.ext.jooqflyway.jobactivity;
+
+import java.io.InputStream;
+import java.security.KeyStore;
+import javax.net.ssl.SSLContext;
+import javax.net.ssl.SSLSocketFactory;
+import javax.net.ssl.TrustManagerFactory;
+
+public final class RdsUtils {
+    /**
+     * Setting as a constant as it is not critical.
+     */
+    private static final String TRUST_STORE_PASSWORD = "titus123";
+
+    public static SSLSocketFactory createRdsSSLSocketFactory() {
+        try {
+            KeyStore trustStore = KeyStore.getInstance(KeyStore.getDefaultType());
+            try (InputStream inputStream = RdsUtils.class.getClassLoader().getResourceAsStream("RDS-2019.truststore")) {
```

Comment:
For security reason, we should allow injecting the truststore file name and the password here. Thus internally we can use a different file, and a different (ideally encrypted) password.

Reply:
We should reference the attached truststure in this repo for running the integration tests only.

Reply:
will fix this in the following pr, finally able to test this. was fighting gradle builds. definitely needs more work here